src/modules/Lan.rb

```diff
@@ -1106,6 +1111,34 @@ def refresh_lan_items
       LanItems.SetModified
     end
 
+    # Returns the routing summary
+    #
+    # @param mode [String,Symbol] Summary mode
+    # @return [String]
+    def routing_summary(mode)
+      config = find_config(:yast)
+      presenter = Y2Network::Presenters::RoutingSummary.new(config.routing)
+      presenter.text(mode: mode.to_sym)
+    end
+
+    # Returns the system configuration
+    #
+    # Just a convenience method.
+    #
+    # @return [Y2Network::Config]
+    def running_config
```

Comment:
why not `system_config` ? running indecates for me runtime configuration but we read only system one, not actually running one.